src/components/dictionaryConcepts/components/AnswersRow.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import autoBind from 'react-autobind';
+import PropTypes from 'prop-types';
+
+class AnswersRow extends Component {
+  static propTypes = {
+    newRow: PropTypes.string.isRequired,
+    addDataFromAnswer: PropTypes.func.isRequired,
+    removeAnswer: PropTypes.func.isRequired,
+    removeDataFromRow: PropTypes.func.isRequired,
+    existingConcept: PropTypes.func.isRequired,
+  }
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: this.props.newRow,
+      answer: '',
+    };
+    autoBind(this);
+  }
+
+  componentDidMount() {
+    if (this.props.existingConcept) {
+      this.updateState();
+    }
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (prevState !== this.state) {
+      this.sendToTopComponent();
+    }
+  }
+
+  updateState() {
+    const { newRow } = this.props;
+    this.setState({
+      ...this.state,
+      uuid: newRow.uuid,
+      answer: newRow.answer,
+    });
+  }
+
+  handleChange(event) {
+    const {
+      target: { value, name },
+    } = event;
+
+    this.setState(() => ({ [name]: value }));
+    this.sendToTopComponent();
+  }
+
+  sendToTopComponent() {
+    this.props.addDataFromAnswer(this.state);
+  }
+
+  handleRemove(event, id) {
+    this.props.removeAnswer(event, id);
```

Comment:
Please use destructuring prop assignment across all your changes

Reply:
Thanks for noticing that. Will work on it.